python/data_structures/linked_list.py

```diff
@@ -1,3 +1,6 @@
+from email import header
```

Comment:
Looks like your editor may have auto-imported this on accident. I have the same problem and don't really know a great fix. Just have to keep an eye out.